packages/range/range.0.3/opam

```diff
@@ -0,0 +1,18 @@
+opam-version : "2.0"
+synopsis: "Fold on integer range"
+description: """
+Enable folding on integer range with split capacities
+for making distributed computing easier with libraries
+like functory
+"""
+maintainer : "Aldrik KLEBER <contact@aldrik.net>"
+authors : "Aldrik KLEBER"
+license : "gpl-v3"
+homepage : "https://github.com/aldrikfr/range"
+dev-repo: "git+https://github.com/aldrikfr/range"
+bug-reports :"https://github.com/aldrikfr/range/issues"
+depends: [
+"ocaml" {>= "4.07.0"}
```

Comment:
Why such a strict bound on ocaml? I did not seem to find any code that requires it